www/src/data/sidebars/doc-links.yaml

```diff
@@ -296,6 +294,8 @@
       link: /docs/prpl-pattern/
     - title: Querying data with GraphQL
       link: /docs/querying-with-graphql/
+    - title: Fetching client data
```

Comment:
What is the more common way of saying this phrase? "fetching client data" or "client data fetching"? It isn't a huge deal, though I think it'd be nice if the title and URL match. Just don't know which one is more common usage. In a cursory google search, looks like "data fetching" is common. This would slightly break the visual pattern of having an "-ing" verb at the beginning of every guide, though technically, it would still grammatically match that pattern, because "client data fetching" is just a _kind_ of fetching. The "client" and "data" are modifiers to "fetching"...if that makes any sense at all!

Reply:
I would tend to agree, and this was my original change, but it's wholly inconsistent with the rest of the sidebar https://github.com/gatsbyjs/gatsby/pull/10932#discussion_r247081938

Reply:
Hm yeah. I like the original change, not sure who has the final call here. Probably @marcysutton!